README.md

```diff
@@ -13,21 +12,45 @@ Teams spend significant time building parts of the design system (colors, variab
 [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
 ![license](https://img.shields.io/github/license/mashape/apistatus.svg)
 
+:star: We appreciate your star, it helps!
+
+## Key features & benefits
+### Unopionated
+These days, there are a million and one great ways to build your next Web site or application. So, we have made being unopinionated the center of every decision made in FAST-DNA. 
```

Comment:
I think it would be more direct to say "There are a million and one great ways to build your next website or application. To fit in with whichever system or technology you choose, the center of every decision made in FAST-DNA is to create unopinionated code."

Reply:
I tweaked yours a bit and landed on "There are a million and one great ways to build your next website or application. To support the technologies you choose, creating unopinionated code is at the center of every decision we make in FAST-DNA."